theme/src/resources/templates/request_page.hbs

```diff
@@ -324,5 +326,17 @@
 		}
 
 		Liferay.hasWatcherPermission(window.HelpCenter.user.tags, {{request.id}}, toggleLargeFileAttachment);
+
+		const orgNameField = document.getElementById('request_organization_id');
+
+		let orgName = '-';
+
+		if (orgNameField && orgNameField.selectedIndex) {
+			orgName = orgNameField.options[orgNameField.selectedIndex].text;
+		}
+
+		const orgNamePlaceholder = document.getElementById('orgNamePlaceholder');
+
+		orgNamePlaceholder.innerHTML = orgName;
```

Comment:
We should check to make sure `orgNamePlaceholder` is not null or undefined.

Reply:
`orgNamePlaceholder` is a `div` in the template. Even if a Zendesk Curlybar helper were to fail, that div would still get rendered. Do we still need to null check?

Reply:
I would do it as a fail safe for the JS side just in case someone removed it accidentally or something happened further down the line which they were unaware of the ramifications.  Always better to be safe than sorry with a simple check.

Reply:
https://github.com/rdai10/zendesk/pull/73

Reply:
Thanks @rdai10!